src/main/resources/configCivcraft.yml

```diff
@@ -9487,7 +9472,7 @@ recipes:
     input:
       emeralds:
         material: EMERALD
-        amount: 5
+        amount: 10
```

Comment:
To bring them overall more in line with other recipes, from 5 to 10 pretty much isnt a difference. I still reduced the higher tiers for those, so it's still getting cheaper overall.